app/components-react/sidebar/SideNav.m.less

```diff
@@ -1,147 +1,217 @@
+.menu-container {
+  flex: 1 1 auto;
+  border-right: 0px !important;
 
-    &.studio-dropdown-active {
-      color: var(--teal);
+  :global(.ant-menu-item-icon) {
+    i {
+      font-size: 18px !important;
     }
   }
-}
 
-.studio-tabs {
-  max-height: 230px;
-  position: absolute;
-  top: 46px;
-  left: 0;
-  overflow-y: auto;
-  background-color: var(--studio-tabs);
-  z-index: 10;
-  border-radius: 15px;
-
-  &::-webkit-scrollbar {
-    display: none;
+  &.menu-wrapper {
+    :global(.ant-menu-item:hover),
+    :global(.ant-menu-submenu:hover) {
+      i {
+        transition: all 0.2s ease-in-out;
+        transform: scale(1.2, 1.2);
+      }
+    }
   }
-}
 
-.main-cell {
-  .cell;
+  :global(li.ant-menu-item.ant-menu-item-selected),
+  :global(li.ant-menu-submenu.ant-menu-item-selected) {
+    background: none;
 
-  svg {
-    width: 16px;
-    height: 16px;
+    &::after {
+      display: none;
+    }
   }
 
-  &.active {
-    i, svg {
-      transform: scale(1.2, 1.2);
-      color: var(--teal);
-      fill: var(--teal);
+  :global(li.ant-menu-item),
+  :global(li.ant-menu-submenu) {
+    background: none;
+    color: var(--paragraph);
+    height: unset !important;
+    min-height: 30px !important;
+    line-height: normal !important;
+    margin: 10px 0 !important;
+
+    i {
+      color: var(--paragraph);
+    }
+
+    :active,
+    :link,
+    :focus,
+    :visited {
+      background: none;
+    }
+
+    :hover {
+      color: var(--title) !important;
     }
   }
 
-  &.disabled {
-    &:hover {
-      cursor: not-allowed;
-      background: transparent;
+  &.closed {
+    :global(.ant-menu-item:hover),
+    :global(.ant-menu-submenu:hover) {
+      i {
+        transition: all 0.2s ease-in-out;
+        transform: scale(1.2, 1.2);
+      }
+    }
+
+    :global(span.ant-menu-title-content) {
+      display: none !important;
+    }
+  }
+
+  &.open {
+    :global(li.ant-menu-item),
+    :global(li.ant-menu-submenu) {
+      overflow: unset !important;
+      text-overflow: unset !important;
+      overflow-wrap: break-word !important;
+      padding-bottom: 0px;
+      white-space: normal !important;
     }
   }
 }
 
-.cell.toggle-on {
+.studio-mode {
+  color: var(--icon-active) !important;
+
   i {
-    color: var(--teal);
+    color: var(--icon-active) !important;
   }
 }
 
-.new-badge {
-  background: var(--new-badge);
-  color: var(--new-badge-text);
-  font-size: 9px;
-  position: absolute;
-  right: 2px;
-  top: 2px;
-  padding: 0 2px;
-  line-height: 12px;
-  font-weight: 400;
+.active {
+  background-color: none;
+  border: none;
+  color: var(--icon-active) !important;
+
+  i {
+    color: var(--icon-active) !important;
+  }
 }
 
-.bottom-tools {
-  margin-top: auto;
-  margin-bottom: 12px;
-  flex-shrink: 0;
+.hide-submenu {
+  display: none;
+}
 
-  .cell {
-    height: 35px;
-    font-size: 14px;
+.ant-menu-inline-collapsed-tooltip {
+  display: none;
+}
+
+.side-nav.ant-slider.ant-slider-handle.ant-tooltip-open {
```

Comment:
i imagine these are either old/irrelevant or need to be made `:global`?

Reply:
Refactored